app/pb_kits/playbook/pb_nav/_item.jsx

```diff
@@ -1,34 +1,31 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
+import React, { Component } from 'react';
 
-const propTypes = {
-  text: PropTypes.string.isRequired,
-  link: PropTypes.string,
-  active: PropTypes.bool
-};
+type NavItemProps = {
+  text: String,
+  link: String,
+  active?: Boolean,
+  children: Array<React.ReactChild> | React.ReactChild,
```

Comment:
You could use `React.Node` [which more accurately defines `children`](https://flow.org/en/docs/react/children).